src/views/StandaloneOfficeAddressFiling.vue

```diff
@@ -584,33 +603,25 @@ export default {
           legalName: this.entityName
         }
       }
-      if (this.hasFilingCode(this.feeCode) && this.addresses) {
-        if (this.addresses.recordsOffice) {
```

Comment:
Again, checking "this.addresses" and "this.addresses.recordsOffice" may have be safety checks before de-referencing them below on line 594-595 etc. But now that "this.addresses" is always an object (never null), this is no longer needed.

Anyway, if we have a filing code then addresses must exist.